keps/sig-auth/2579-psp-replacement/README.md

```diff
@@ -500,7 +500,11 @@ role must be unset. Spec:
 **Non-root Groups** - (restricted) This optional constraint will be omitted from the initial
 implementation.
 
-**Seccomp** - (restricted) Allow anything except `unconfined`. Same reasoning as AppArmor.
+**Seccomp**
+- (restricted) Must be set to anything except `unconfined`. Same reasoning as AppArmor. If the default
```

Comment:
is there a default policy that is reasonable to set that is "universally" supported?

Reply:
I think we will have to audit what's used across CRI-O/containerd/docker to see if we can come down to one default policy.

Reply:
Yeah, ["RuntimeDefault"](https://github.com/kubernetes/kubernetes/blob/master/staging/src/k8s.io/api/core/v1/types.go#L3367). There's an open KEP to make it the actual default: https://github.com/kubernetes/enhancements/pull/2414